recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@
 
 {% set intel_ch = "https://software.repos.intel.com/python/conda" %}
 # use this if our build script changes and we need to increment beyond intel's version
-{% set dstbuildnum = '1' %}
+{% set dstbuildnum = '0' %}
```

Comment:
Must be increased, since other versions was not changed

Reply:
Just updated other package info - is it needed to be set to 2? This parameter is unclear to me

Reply:
Thank you! Please also update `openmp_version`. In other words, this may be reset only if all version where increased. If some version stayed the same then `dstbuildnum` must be increased. This needed to prevent this scenario:

- Imagine this update was merged, and `intel-openmp` was not changed. It already pushed as  `version 2024.2.0, build 982 (981 +1)`. So it will be pushed as same version but 981 (981 + 0) build. It will be okay for now, since it did not affect latest build.
- However, imagine after that change merged somebody updating `intel-openmp` with repacking change, keeping version the same. At that PR `dstbuildnum` will be increased, but version will be the same: `version 2024.2.0, build 982 (981 +1)`. However this change was already uploaded, so after merge publication will fail.

It is confusing at first place, but is even more confusing because we have same build number counter for multiple packages and versions.